### PR TITLE
Vallox switch platform

### PR DESCRIPTION
**Pull request by yozik04**
## Proposed change

Added description about new **switch** platform.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/75857
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/vallox.markdown [hunk 1]
```diff
@@ -23,7 +25,9 @@ The `vallox` integration lets you control any Vallox ventilation unit that is su
 
 The **fan** platform of this integration allows you to turn on/off the complete unit via the toggle switch and to select a ventilation profile.
 
-Also, there is a **sensor** platform that exposes a number of relevant metrics like fan speed, various air temperatures and humidity.
+The **sensor** platform that exposes a number of relevant metrics like fan speed, various air temperatures and humidity.
+
+The **switch** platform allows to lock heat recovery bypass.
```
Comment (frenck) [inline source/_integrations/vallox.markdown]:
```suggestion
The **sensor** platform exposes a number of relevant metrics like fan speed, various air temperatures, and humidity.

The **switch** platform allows locking heat recovery bypass.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by yozik04** [comment]
Thank you @frenck

**Comment by yozik04** [comment]
@frenck I do not see how to get rid of "Changes requested". I have applied these.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by yozik04** [comment]
@frenck Maybe you can do the final review of the parent PR: https://github.com/home-assistant/core/pull/75857
It sits there for quite a long time already.

**Pull request merged**